resources/views/setup.blade.php

```diff
@@ -26,22 +26,22 @@
             <h4>Database Configuration</h4>
 
             <p>Database Host:</p>
-            <input type='text' class='form-control' name='db:host' value='localhost'>
+            <input type='text' class='form-control' name='db:host' value='{{ $db_host }}'>
```

Comment:
Can we use camel case as per PSR2 please.

Reply:
Certainly, I prefer camel casing but thought you guys used underscores when I saw this:

![selection_801](https://user-images.githubusercontent.com/5709838/38135740-bca2df0a-3411-11e8-9e63-eb09b058d576.png)